docker-compose.prod.yml

```diff
@@ -3,8 +3,12 @@ services:
   production:
     image: production
     build: .
+    restart: always
     ports: 
       - '1323:1323'
+    security_opt:
+      - no-new-privileges
+    read_only: true
```

Comment:
Verify it still works when using an `ro` volume - oftentimes programs will like to write temp files

Reply:
Will do.

Reply:
Yep it is ro.